site/docs/v1/tech/guides/multi-factor-authentication.adoc

```diff
@@ -536,8 +537,62 @@ If you have other MFA methods that you'd like to use with FusionAuth, you have a
 
 include::docs/v1/tech/guides/_mfa-migration.adoc[]
 
+== Forcing MFA
+
+There are times when you might want to force a user to provide an additional factor of authentication before they ever get access to your application.
+For example, you might require MFA when a user logs into an accounting application, but not when they log in to a customer support application.
+
+Or you might want to force every user with a certain role to provide an additional factor every time they log in.
+For example, super admins might need to provide a second factor every time they log in.
+
+In FusionAuth, if a user has one or more additional factors configured for their account, they will be forced to provide that second factor on login.
+Once someone has provided a second factor, they will not have to provide it again until a certain amount of time has passed.
+This duration is configurable at the tenant level, and is set by the `twoFactorTrustIdTimeToLiveInSeconds` setting.
+
+=== Forcing a User to Provide an Additional Factor Before Accessing an Application
+
+You may want to force a user to perform MFA before they can access a sensitive application, such as the accounting application mentioned above.
+
+One way to do that is to add link:/docs/v1/tech/guides/multi-factor-authentication#step-up-auth[step up authentication] to your application.
+Each time the user accesses the application, perform a step up, which will force them to provide an additional factor.
+
+An alternate option is to deny access if the user has configured any MFA method, since a user with an MFA method enabled will have used it on log in.
```

Comment:
```suggestion
An alternate option is to deny access if the user has not configured any MFA method, since a user with an MFA method enabled will have used it on log in.
```